optimum/onnxruntime/configuration.py

```diff
@@ -277,10 +277,12 @@ class QuantizationConfig:
     qdq_op_type_per_channel_support_to_axis: Dict[str, int] = field(
         default_factory=lambda: ORT_DEFAULT_CHANNEL_FOR_OPERATORS
     )
+    disable_shape_inference: bool = False
```

Comment:
Can you also add a description of the attribute in the docstring? Thx!

Reply:
done!